src/providers/DevicesProvider/AudioInputProvider.tsx

```diff
@@ -66,6 +83,7 @@ const AudioInputProvider: React.FC = ({ children }) => {
           (device) => device.deviceId === selectedInputRef.current
         );
 
+        let nextInput: string = '';
```

Comment:
You should not need the assignment here, but perhaps TypeScript's analysis is weak. If so (try it), start with `default`.

Reply:
Tried it without the assignment, got an error like `Variable 'nextInput' is used before being assigned. `
Set the value to `default`.